src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -1605,32 +1606,32 @@ private async Task<TokenValidationResult> ValidateJWEAsync(JsonWebToken jwtToken
             }
         }
 
-        private static JsonWebToken ValidateSignatureUsingDelegates(string token, TokenValidationParameters validationParameters, BaseConfiguration configuration)
+        private static JsonWebToken ValidateSignatureUsingDelegates(JsonWebToken jsonWebToken, TokenValidationParameters validationParameters, BaseConfiguration configuration)
         {
             if (validationParameters.SignatureValidatorUsingConfiguration != null)
             {
-                var validatedToken = validationParameters.SignatureValidatorUsingConfiguration(token, validationParameters, configuration);
+                var validatedToken = validationParameters.SignatureValidatorUsingConfiguration(jsonWebToken.EncodedToken, validationParameters, configuration);
```

Comment:
>jsonWebToken [](http://example.com/codeflow?start=95&length=12)

need any null checks on `jsonWebToken`?

Reply:
this is from dev introduced here: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2064/files#diff-e655d62353a35ce79c5d87080d5f31d3bcd83c6f16e6b0690265f32c7d6ffe7b